### PR TITLE
Clarify Local Package Repository Section

### PR DESCRIPTION
**Pull request by ab7pa**
Clarify the Local Package Repository instructions in the How-To section.

### CODE REVIEW

**Comment by ab7pa** [comment]
Let me know if any changes are required.  Many thanks!
Steve  AB7PA

**Comment by ab7pa** [comment]
Okay, I think this PR is good to go.  THX

File: arednHow-toGuides/local-package-repo.rst [hunk 1]
```diff
@@ -7,46 +7,46 @@ There may be cases where your mesh nodes have no way to access the AREDN |trade|
...
+Your package server must be connected to the mesh as a host on your local node's LAN network, using a node that also has Internet access via its WAN interface. You should add this host to the node's *DHCP Reservation List*. You do not need to add the package host to the *Advertised Services List* of the node to which it is connected. The package server should be given a hostname that is unique on your mesh, typically prefixed with the callsign of the server owner. You can use any operating system platform you desire *(Windows, Linux, Mac),* as long as it has the ability to function as a web server. The following are the two main tasks required of the local package server:
```
Comment (ae6xe) [inline arednHow-toGuides/local-package-repo.rst:10]: The context and assumption is that this mesh network does not have any advertised gateways.  Consequently, the admin following these directions may be concerned and need further explanation on this statement, "must be connected to the mesh as a host on your local node's LAN network, using a node that also has Internet access via its WAN interface."    

The admin may not realize, "The reason this node is connected to the internet is to allow the web server to download updated files from the AREDN internet server, but at the same time the internet connection is not advertised or allowed use by other nodes or devices on the mesh network"

Comment (ab7pa) [reply arednHow-toGuides/local-package-repo.rst]: Excellent clarification, Joe.  Changes made and committed.  Many thanks!

**Pull request merged**